deploy/chart/templates/0000_50_olm_08-catalog-operator.deployment.yaml

```diff
@@ -17,6 +17,13 @@ spec:
       labels:
         app: catalog-operator
     spec:
+      securityContext:
+        runAsNonRoot: true
+        seccompProfile:
+          type: RuntimeDefault
+        {{- if eq .Values.installType "upstream" }}
+        runAsUser: {{ .Values.package.securityContext.runAsUser }}
+        {{- end }}
```

Comment:
```suggestion
```
Same. We cannot use RunAsUser at all,
That is ONLY required because the Dockerfile image is not defining the user when should be.
So, what is the image used here? 

Another option, in this case, is to ensure enforcement as a baseline or privileged and not set RunAsNonRoot either.

Reply:
Nah we should keep security as nerfed as possible. I don't think the controllers need anything above restricted.